components/rhs_thread/rhs_thread.tsx

```diff
@@ -297,6 +293,16 @@ export default class RhsThread extends React.Component<Props, State> {
         });
     }
 
+    private handlePostCommentResize = (): void => {
+        const postCreateContainerHeight = 190;
+        const createContainerHeight = this.postCreateContainerRef.current?.getBoundingClientRect().height;
+
+        if (createContainerHeight !== postCreateContainerHeight) {
```

Comment:
We can remove this check. `AutosizeTextarea` already makes sure that the height changed before calling this.